docs/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.md

```diff
@@ -448,7 +448,7 @@ The value you use for this CSS rule should identify one of the fontello icons in
 
 It is not possible at this time to set an icon for your add-on package control panels without including CSS in your package.
 
-For documentation on how to use it in your own add-ons see https://training.plone.org/5/mastering-plone-5/registry.html.
+For documentation on how to use it in your own add-ons see {ref}`training:plone5-registry-controlpanel-label`.
```

Comment:
Actually this label does not resolve.

https://deploy-preview-1463--6-docs-plone-org.netlify.app/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.html#control-panel

What should it be?

Reply:
Perhaps this?

```
{doc}`training:mastering-plone-5/registry`
```